docs/run/get-backend-information.ipynb

```diff
@@ -101,241 +146,142 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Configuration\n",
+    "## Static backend information\n",
     "\n",
-    "The configuration is where you'll find data about the static setup of the device, such as its name, version, the number of qubits, and the types of features it supports.\n",
-    "\n",
-    "Let's build a description of our backend using information from the `backend`'s config."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {
-    "execution": {
-     "iopub.execute_input": "2023-08-25T18:26:00.182928Z",
-     "iopub.status.busy": "2023-08-25T18:26:00.180031Z",
-     "iopub.status.idle": "2023-08-25T18:26:00.800954Z",
-     "shell.execute_reply": "2023-08-25T18:26:00.800060Z"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "from qiskit.providers.fake_provider import FakeHanoi\n",
+    "Some information about a backend does not change regularly, such as its name, version, the number of qubits it has, and the types of features it supports. This information is available as attributes of the `backend` object.\n",
     "\n",
-    "backend = FakeHanoi()"
+    "The following cell builds a description of a backend."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {
-    "execution": {
-     "iopub.execute_input": "2023-08-25T18:26:00.805066Z",
-     "iopub.status.busy": "2023-08-25T18:26:00.804526Z",
-     "iopub.status.idle": "2023-08-25T18:26:00.812013Z",
-     "shell.execute_reply": "2023-08-25T18:26:00.811250Z"
-    }
-   },
+   "execution_count": 5,
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "This backend is called fake_hanoi, and is on version 1.0.18. It has 27 qubits. It supports OpenPulse programs. The basis gates supported on this device are ['id', 'rz', 'sx', 'x', 'cx', 'reset'].\n"
+      "Name: ibm_lagos\n",
+      "Version: 2\n",
+      "No. of qubits: 7\n",
+      "\n"
      ]
     }
    ],
    "source": [
-    "config = backend.configuration()\n",
+    "backend = service.backend(\"ibm_lagos\")\n",
     "\n",
-    "# Basic Features\n",
-    "print(\"This backend is called {0}, and is on version {1}. It has {2} qubit{3}. It \"\n",
-    "      \"{4} OpenPulse programs. The basis gates supported on this device are {5}.\"\n",
-    "      \"\".format(config.backend_name,\n",
-    "                config.backend_version,\n",
-    "                config.n_qubits,\n",
-    "                '' if config.n_qubits == 1 else 's',\n",
-    "                'supports' if config.open_pulse else 'does not support',\n",
-    "                config.basis_gates))"
+    "print(\n",
+    "    f\"Name: {backend.name}\\n\"\n",
+    "    f\"Version: {backend.version}\\n\"\n",
+    "    f\"No. of qubits: {backend.num_qubits}\\n\"\n",
+    ")"
```

Comment:
I'm ignoring information from `backend.configuration()` since that doesn't exist in Qiskit's `BackendV2`.